new code

```diff
@@ -0,0 +1,163 @@
+import java.util.*;
+public class tictactoenew
```

Comment:
class name should be camelcase. 
Everything in java in camelcase
so it should be like public class `TicTacToe`
and no new is needed. What is new here ?